tests-e2e/fixture/fixture.go

```diff
@@ -601,6 +601,11 @@ func GetKubeClient(config *rest.Config) (client.Client, error) {
 		return nil, err
 	}
 
+	err = appstudiosharedv1.AddToScheme(scheme)
```

Comment:
Same here, think we've already got this covered :detective:

Reply:
Fixed.  Duplicate was a consequence of rebasing on top of the change which moved the api into a separate repo.